ssl/record/ssl3_record.c

```diff
@@ -381,9 +389,9 @@ int ssl3_get_record(SSL *s)
         goto f_err;
     }
 #ifdef SSL_DEBUG
-    printf("dec %d\n", rr->length);
+    printf("dec %ld\n", rr->length);
```

Comment:
[Allegedly](https://blogs.msdn.microsoft.com/vcblog/2014/06/03/visual-studio-14-ctp/#div-comment-77743) even Microsoft has finally caught up with the tail end of the 20th century.  How feasible is it to just use `%zu` from OpenSSL $NEXT?